.claude/commands/ship.md

```
@@ -0,0 +1,16 @@
+Run the full pre-ship sequence for: $ARGUMENTS
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)